src/platform/forms-system/src/js/review/SubmitController.jsx

```diff
@@ -60,7 +60,7 @@ class SubmitController extends React.Component {
 
     // Validation errors in this situation are not visible, so we’d
     // like to know if they’re common
-    const { isValid, errors } = isValidForm(form, pagesByChapter);
+    const { isValid, errors } = isValidForm(form, pageList);
```

Comment:
From what I could tell, the actual pages inside `pagesByChapter` are the same as `pageList` except that the latter has `appStateData` from a `mapStateToProps` in `RoutedSavableReviewPage`.